### PR TITLE
Master

### PR DESCRIPTION
**Pull request by phill2mj**
Pull changes adding cards/monster cards framework

### CODE REVIEW

File: src/app/app.component.scss [hunk 1]
```diff
@@ -16,6 +16,7 @@
   .card-band {
     grid-area: card-band;
     border-bottom: solid 0.5px #cccccc;
+    display: flex;
```
Comment (ninebolt) [inline src/app/app.component.scss]: Is this needed? Wouldn't it be better to set `display: flex` on the root of `card-deck`?

File: src/app/containers/card-deck/card-deck.component.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+@Component({
+    selector: 'card-deck',
+    styleUrls: ['card-deck.component.scss'],
+    template: `
+        <div *ngFor="let c of monsterCards" class="card-container">
+            <monster-card (click)="cardClick(c)"
+                [ngClass]="{
+                    'flipped flip-to-front': c.facing === 'up',
+                    'send-to-back': c.facing === 'down',
+                    'shuffle': this.shuffleDeck === true
```
Comment (ninebolt) [inline src/app/containers/card-deck/card-deck.component.ts]: would it be `this.shuffleDeck` or `shuffleDeck`?

File: src/app/containers/card-deck/card-deck.component.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        this.sortInOrder(this.monsterCards, 'order');
+
+        // add in a test shuffle card
+        this.monsterCards[5].shuffle = true;
+
+        // console.log(this.monsterCards);
+    }
+
+    private cardClick(card) {
+        if (card.shuffle === true) {
```
Comment (ninebolt) [inline src/app/containers/card-deck/card-deck.component.ts]: The deck should not shuffle if a card with the shuffle flag shows up. The deck will be shuffled at the end of the round. On card-clicks, you should just be checking if the revealed card is a 'shuffle-trigger' card and update a local variable to be referenced later.

Comment (phill2mj) [reply src/app/containers/card-deck/card-deck.component.ts]: 👍

File: src/app/containers/card-deck/card-deck.component.ts (hunk 2 above)
Comment (phill2mj) [reply src/app/containers/card-deck/card-deck.component.ts]: N/A, but honestly I'm not sure

File: src/app/app.component.scss (hunk 1 above)
Comment (phill2mj) [reply src/app/app.component.scss]: yep, but that's still placeholder styling. removed anyway.

**Comment by phill2mj** [comment]
[now lookie here, sir](https://github.com/ninebolt/gloom-assistant/pull/2/commits/c902c1481f611aff10864a57e33762bb1da9722b)

**Pull request merged**